toolchain/lower/file_context.h

```diff
@@ -28,6 +31,13 @@ class FileContext {
     int32_t column_number;
   };
 
+  // Describes a specific function's body fingerprint
+  struct SpecificFunctionFingerprint {
+    llvm::BLAKE3Result<32UL> function_common_fingerprint;
+    llvm::BLAKE3Result<32UL> function_specific_fingerprint;
```

Comment:
Do these need to be marked `UL`?

```suggestion
    llvm::BLAKE3Result<32> function_common_fingerprint;
    llvm::BLAKE3Result<32> function_specific_fingerprint;
```